BricksAndTiles/Source/CollectionView/Sections/GridCollectionSection.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  GridCollectionSection.swift
+//  BricksAndTiles
+//
+//  Created by Tulio Bazan on 24/09/20.
+//  Copyright © 2020 TucoBZ. All rights reserved.
+//
+
+import UIKit
+
+public final class GridCollectionSection: CollectionViewSection {
+
+    public let spacingBetweenItems: ItemSpacing
+    public let sectionInsets: UIEdgeInsets
+
+    private let numberOfItemsPerRow: UInt
```

Comment:
Shouldn't be `Int`?
```suggestion
    private let numberOfItemsPerRow: Int
```

Reply:
`UInt` guarantees it will be positive, and not less than `zero`. Maybe you will never want this property be negative.

Reply:
Nice! Didn't know about this one 😅